src/views/Records/Records.vue

```diff
@@ -122,13 +122,13 @@ export default {
     });
   },
   destroyed() {
+    window.scrollTo(0, 0);
```

Comment:
why are we still using window.scrollTo instead of vue-scrollTo ?

Reply:
due to I need the window.top.scrollY for some calculation in records but v-scrollTo is able to give body specifications not window object. I need window object in some cases but I will investigate to see if I can unify window scrolls into vue-scrollTo..